ios/RNGestureHandlerButtonComponentView.mm

```diff
@@ -0,0 +1,48 @@
+#ifdef RN_FABRIC_ENABLED
```

Comment:
It is added to reduce the final code size on paper? If so, should we do the same but inverted for Fabric code?

Reply:
It implements `RCTRNGestureHandlerButtonViewProtocol` from codegen. This way it doesn't interfere with compilation on Paper where it's not used.

Reply:
Another option would be to manually exclude these files in Podspec, either by using `*.{h,m}'` pattern for `source_files` or alternatively via `exclude_files` property, but we prefer the preprocessor conditionals.